python/tskit/trees.py

```diff
@@ -8592,6 +8592,276 @@ def genetic_relatedness_vector(
         )
         return out
 
+    def _genetic_relatedness_vector_node(
+        self,
+        arr: np.ndarray,
+        indices: np.ndarray,
+        mode: str,
+        centre: bool = True,
+        windows = None,
+        ) -> np.ndarray:
+        x = arr - arr.mean(axis=0) if centre else arr
+        x = self.genetic_relatedness_vector(
+            W=x, windows=windows, mode=mode, centre=False, nodes=indices,
+        )[0]
+        x = x - x.mean(axis=0) if centre else x
+
+        return x
+
+    def _genetic_relatedness_vector_individual(
+        self,
+        arr: np.ndarray,
+        indices: np.ndarray,
+        mode: str,
+        centre: bool = True,
+        windows = None,
+        ) -> np.ndarray:
+        ij = np.vstack(
+            [
+                [n, k]
+                for k, i in enumerate(indices)
+                for n in self.individual(i).nodes
+            ]
+        )
+        samples, sample_individuals = (
+            ij[:, 0],
+            ij[:, 1],
+        )  # sample node index, individual of those nodes
```

Comment:
Putting comments at the end of lines is causing them to get broken by Black. Better to put the comments on the line immediately above.